contracts/account/extensions/AccountSignerERC7702.sol

```diff
@@ -3,12 +3,13 @@
 pragma solidity ^0.8.20;
 
 import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
-import {AccountCore} from "../AccountCore.sol";
+import {AbstractSigner} from "../../utils/cryptography/AbstractSigner.sol";
 
 /**
- * @dev {Account} implementation whose low-level signature validation is done by an EOA.
+ * @dev Implementation of {AbstractSigner} for implementation of ERC-7702 accounts whose low-level signature 
+ * validation is done by an EOA.
  */
-abstract contract AccountSignerERC7702 is AccountCore {
+abstract contract AccountSignerERC7702 is AbstractSigner {
```

Comment:
Perhaps we might just name this `SignerERC7702`

```suggestion
abstract contract SignerERC7702 is AbstractSigner {
```

And move it to the `cryptography` folder (?)

Reply:
Yes this is a signer, but unlike the other ones, I don't think this one makes sens outside of an account. That is why I kept the name and location. But I you think it makes more sens to rename and move it I wouldn't oppose it.

Reply:
I'm trying to thing about the developer experience of this. Most would understand that ERC-7702 enables signature for EOAs and these use ECDSA, so I'd be worried they'd start from a SignerECDSA instead where is kinda weird that you'll need to initialize by calling `_initializeSigner`.

I'm leaning towards moving it so I'll push a commit